README.md

```diff
@@ -1,3 +1,4 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+In order to run the project locally, create two files for the databases environment variable: '.env.test' and '.env.development'.
```

Comment:
Nice start! It would be beneficial here to state what is actually inside the files aswell as the format eg add the database names